Speculator/Speculator/ViewModels/MainWindowViewModel.cs

```diff
@@ -53,6 +53,17 @@ public MainWindowViewModel()
         Settings.PropertyChanged += (_, _) => OnSettingsChanged(true);
         OnSettingsChanged(false);
 
+        if (args.Length > 0)
```

Comment:
Nice one! I might suggest adding a method to ZxFileIo like this:
```
    public static bool IsInstantLoadSupported(FileInfo fileInfo)
    {
        switch (fileInfo.Extension.ToLower())
        {
            case ".bin":
            case ".sna":
            case ".scr":
            case ".z80":
                return true;
            default:
                return false;
        }
    }

```
...then we could call that instead of checking for a `.bin` extension.
That would open up the feature to all 'instant loading' file formats.

Reply:
done